### PR TITLE
RULEAPI-675: When searching for a rule-id the results should start with the corresponding RSPEC

### PR DESCRIPTION
**Pull request by csaba-sagi-sonarsource**

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html prefix="og: https://ogp.me/ns#" lang="en">
```
Comment (vilchik-elena) [inline frontend/public/index.html:2]: @csaba-sagi-sonarsource should it be in a different PR?

Comment (csaba-sagi-sonarsource) [reply frontend/public/index.html:2]: Yep, my bad, I`ll move it to a separate PR.

**Review by vilchik-elena — APPROVED** [review]
@csaba-sagi-sonarsource I refactored your implementation, let me know what you think
